docs/webapp.adoc

```diff
@@ -427,7 +455,7 @@ Just like before, we get react/react-dom from a CDN, provide a pinch of styling
         <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.production.min.js"></script>
 
         <!--1-->
-        <script data-th-src="${portal.serviceUrl({'_service=react4xp-client'})}" src=""></script>
+        [(${bodyEnd})]
```

Comment:
Here and other places that use this kind of bodyEnd insertion:

The name `bodyEnd` only makes sense inside the XP controller, and that's not in focus here, we're trying to use it as little as possible. (And it's not even inserted in the bodyEnd position in the webapp.html, more like the middle of the HEAD 😛 ...) 

I think it would be better with a name that describes what it is in the HTML context, for example `dependencies`?